services/frontend-service/src/ui/Pages/Environments/EnvironmentsPage.tsx

```diff
@@ -13,17 +13,30 @@ You should have received a copy of the MIT License
 along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
 
 Copyright 2023 freiheit.com*/
-import { useEnvironmentNames } from '../../utils/store';
-import { EnvironmentCard } from '../../components/EnvironmentCard/EnvironmentCard';
+import { useEnvironmentGroups, useEnvironmentNames } from '../../utils/store';
+import { EnvironmentCard, EnvironmentGroupCard } from '../../components/EnvironmentCard/EnvironmentCard';
 
 export const EnvironmentsPage: React.FC = () => {
+    const envsGroups = useEnvironmentGroups();
     const envs = useEnvironmentNames();
-
+    const useGroups = envsGroups.length < envs.length;
```

Comment:
```suggestion
    const useGroups = envsGroups.length != envs.length;
```
The current expression may be a bit misleading since we can't have actually have more environment groups than environments